packages/builder/src/pages/builder/app/[application]/_layout.svelte

```diff
@@ -112,23 +223,100 @@
         <VersionModal />
         <RevertModal />
         <Icon name="Play" hoverable on:click={previewApp} />
-        <DeployModal />
+
+        {#if isPublished}
+          <PopoverMenu
+            bind:this={publishPopover}
+            align="right"
+            disabled={!isPublished}
+            dataCy="publish-popover-menu"
+          >
+            <div slot="control" class="icon app-status-icon">
+              <Icon
+                size="M"
+                hoverable
+                name={isPublished ? "Globe" : "GlobeStrike"}
+                disabled={!isPublished}
+              />
+            </div>
+            <Layout gap="M">
+              <Heading size="XS">Your app is live!</Heading>
+              <div class="publish-popover-message">
+                {#if isPublished}
+                  {processStringSync(
+                    "Last Published: {{ duration time 'millisecond' }} ago",
+                    {
+                      time:
+                        new Date().getTime() -
+                        new Date(latestDeployments[0].updatedAt).getTime(),
+                    }
+                  )}
+                {/if}
+              </div>
+              <div class="publish-popover-actions">
+                <Button
+                  warning={true}
+                  icon="GlobeStrike"
+                  disabled={!isPublished}
+                  on:click={unpublishApp}
+                  dataCy="publish-popover-action"
+                >
+                  Unpublish
+                </Button>
+                <Button cta on:click={viewApp}>View App</Button>
+              </div>
+            </Layout>
+          </PopoverMenu>
+        {/if}
+
+        {#if !isPublished}
```

Comment:
We already have a spectrum `Tooltip` component in BBUI, which we often use on icons exactly like this. I think it would be sensible to keep using the existing tooltip here. I understand the `PopoverMenu` usage for the full published status content, but for these tooltips we should just use the existing tooltip. Again as mentioned before, I also think this should be contained in a separate file than the layout. Everything could be placed inside a `PublishPopover` component for example.

I actually added a tooltip setting to the `Icon` component as part of the new design UI work, so I might actually pull that into your branch and we could use it here for the normal tooltips. Would make life much easier!

Reply:
No problem, I've removed the PopoverMenu entirely now. I've reverted to using the Popover component on it's own.